### DISCUSSION TITLE
Can I use TikTok-Api to crawl comments of a video?

### DISCUSSION THREAD

**Discussion by novemberdude** [davidteather/TikTok-Api#447] [Q&A] [answered]
It seems TikTok-Api doesn't support to crawl comments of a video on TikTok? Can you suggest me a way/tools to crawl comments? Thank you!

**Comment by davidteather** [comment, marked as answer]
This package doesn't currently support comment crawling because you must be logged in to view comments #423 it's a planned function of the library at some point in the distant future to support logging in, however, currently, my focus is not on that.

If you want to make API call requests take a look [at this function](https://github.com/davidteather/TikTok-Api/blob/0d9fa3c1d22f78b99854d3c2245025618742fa7b/TikTokApi/tiktokuser.py#L16) you might be able to do something similar for the comment endpoint.
